core/executor/executor.go

```diff
@@ -12,15 +13,15 @@ import (
 type IBlockchain interface {
 	VerifyExpiredAt(expiredAt int64) error
 	VerifyNonce(accountIndex int64, nonce int64) error
-	VerifyGas(gasAccountIndex, gasFeeAssetId int64) error
+	VerifyGas(gasAccountIndex, gasFeeAssetId int64, txType int, gasFeeAmount *big.Int, skipGasAmtChk bool) error
 	StateDB() *sdb.StateDB
 	DB() *sdb.ChainDB
 	CurrentBlock() *block.Block
 }
 
 type TxExecutor interface {
 	Prepare() error
-	VerifyInputs() error
+	VerifyInputs(bool) error
```

Comment:
can we have a name for the param, it's not obvious at the first glance

Reply:
sure, updated